src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: auto;
+  background: rgba(255, 255, 255, 1);
+  width: 200px;
+  border-radius: 10px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-family: $mainFont;
+
+  &__imagine {
+    margin: 32px 19px 0;
+    box-sizing: border-box;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    padding: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__product {
+    margin: 4px 16px 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__value {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 16px 0;
+  }
+
+  &__price {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__sum {
+    margin: 0;
+    height: 18px;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__button {
+    display: inline-block;
+    margin: 16px;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background: rgba(0, 172, 220, 1);
+    text-decoration: none;
+    text-align: center;
+    line-height: 40px;
+    color: rgba(255, 255, 255, 1);
+  }
+
+  &__button:hover {
+    color: rgba(0, 172, 220, 1);
+    background-color: rgba(255, 255, 255, 1);
```

Comment:
Consider using variables for repeating colors

Reply:
done